renter/renter.go

```diff
@@ -331,6 +331,7 @@ func (r *Renter) removeVersion(version *core.Version) {
 func (r *Renter) removeBlock(block *core.Block) error {
 	for _, location := range block.Locations {
 		pvdr := provider.NewClient(location.Addr, &http.Client{})
+		pvdr.AuthorizeRenter(r.privKey, r.Config.RenterId)
```

Comment:
Needs error handling. We can ignore for now since another pull request will make this irrelevant.